documentation/installation.md

```diff
@@ -32,6 +32,12 @@ The Kyverno policy engine runs as an admission webhook and requires a CA-signed
 
 There are 2 ways to configure the secure communications link between Kyverno and the kube-apiserver.
 
+## Kyverno Flags 
```

Comment:
Let's move this to the end of `## Install Kyverno using YAMLs`, the tile should start with `###`.